ivy/functional/frontends/paddle/nn/functional/common.py

```diff
@@ -177,6 +177,21 @@ def unfold(x, kernel_sizes, strides=1, paddings=0, dilations=1, name=None):
     return ivy.stack(folder, axis=2)
 
 
+@to_ivy_arrays_and_back
+@with_supported_dtypes({"2.5.1 and below": ("float32", "float64")}, "paddle")
+def upsample(
+    x,
+    size=None,
+    scale_factor=None,
+    mode="nearest",
+    align_corners=False,
+    align_mode=0,
+    data_format="NCHW",
+    name=None,
+):
+    return interpolate(x, size, scale_factor, mode, align_corners)
```

Comment:
It should be `ivy.interpolate` here.

Reply:
Hi @Dsantra92 thanks for the feedback. I've updated the function to use `ivy.interpolate`